controllers/volumereplication_controller.go

```diff
@@ -64,3 +86,34 @@ func (r *VolumeReplicationReconciler) SetupWithManager(mgr ctrl.Manager, cfg *co
 		For(&replicationv1alpha1.VolumeReplication{}).
 		Complete(r)
 }
+
+func (r VolumeReplicationReconciler) getVolumeHandle(instance *replicationv1alpha1.VolumeReplication) (string, error) {
+	pvcName := instance.Spec.DataSource.Name
+	pvc := &corev1.PersistentVolumeClaim{}
+	err := r.Client.Get(context.TODO(), types.NamespacedName{Name: pvcName, Namespace: instance.Namespace}, pvc)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			r.Log.Info("pvc %q not found", pvcName)
```

Comment:
```suggestion
			r.Log.Info("PVC not found", "PVC Name", pvcName)
```
is the correct syntax.